ToDoApp/ToDoApp/ToDoListViewModel.swift

```diff
@@ -49,25 +49,22 @@ final class ToDoListViewModel: ObservableObject {
     }
     // Handle eddting or creating task when edit or add button is tapped
     private func addEditButtonTapped(mode: CreateTaskViewMode) {
-        let createTaskViewModel: CreateTaskViewModel = CreateTaskViewModel(taskService: taskService, mode: mode) { [weak self] in
+        let createTaskViewModel: CreateTaskViewModel =
+        CreateTaskViewModel(taskService: taskService, mode: mode) { [weak self] in
```

Comment:
Is this new line change required?

Reply:
When I ran swiftlint, it said keep the maximum of around 120 characters on a line, so I break it into 2 lines

Reply:
got it. in that case there is another patter on how to break lines:
```
let createTaskViewModel: CreateTaskViewModel = CreateTaskViewModel(
    taskService: taskService, 
    mode: mode
) { [weak self] in
```

also the type name can be omitted:
```
let createTaskViewModel: CreateTaskViewModel = .init(
    taskService: taskService, 
    mode: mode
) { [weak self] in
```